### ISSUE TITLE
How to install

### ISSUE THREAD

**Issue by smallfish01** [ladybirdweb/faveo-servicedesk-community#8]
Hi,
Sorry for the trouble, how can I install faveo-servicedesk-community for manual? or do you have docker verion?

Thank you.

**Comment by nandagopalnandu** [comment]
HI,
Please find the below link for installing our faveo
https://github.com/ladybirdweb/faveo-helpdesk/wiki
At the time of installation Please replace the clone command with below as the manual contain for faveo helpdesk community edition.
git clone https://github.com/ladybirdweb/faveo-servicedesk-community.git

**Comment by smallfish01** [comment]
Thanks for your quick replied! I just saw faveo-helpdesk wiki and not found the document for faveo-servicedesk-community, is the same for faveo-helpdesk?

**Comment by nandagopalnandu** [comment]
Yes it is. We didnt create a manual for Service desk as it doest show much difference rather than replacing the line as i said above.

**Comment by smallfish01** [comment]
Thanks you so much, I will have a try.

**Comment by fajarwahyu99** [comment]
how to install this i have a problem like this
![image](https://user-images.githubusercontent.com/11350854/38474317-61e4c2e6-3bc7-11e8-8cef-98e8829fec93.png)
